src/fidesctl/api/ctl/migrations/versions/f131a1263a10_add_is_default_to_taxonomy_fields.py

```diff
@@ -0,0 +1,81 @@
+"""Add is_default to taxonomy fields
+
+Revision ID: f131a1263a10
+Revises: f53e04e5b7f5
+Create Date: 2022-08-10 16:25:13.754740
+
+"""
+import sqlalchemy as sa
+from alembic import op
+
+# revision identifiers, used by Alembic.
+revision = "f131a1263a10"
+down_revision = "f53e04e5b7f5"
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.add_column(
+        "data_categories", sa.Column("is_default", sa.BOOLEAN(), nullable=True)
+    )
+    op.drop_index("ix_data_categories_id", table_name="data_categories")
+    op.create_index(
+        op.f("ix_data_categories_id"), "data_categories", ["id"], unique=False
+    )
+    op.add_column(
+        "data_qualifiers", sa.Column("is_default", sa.BOOLEAN(), nullable=True)
+    )
+    op.drop_index("ix_data_qualifiers_id", table_name="data_qualifiers")
+    op.create_index(
+        op.f("ix_data_qualifiers_id"), "data_qualifiers", ["id"], unique=False
+    )
+    op.add_column("data_subjects", sa.Column("is_default", sa.BOOLEAN(), nullable=True))
```

Comment:
did you ever figure out why the index changes were happening or confirm they existed before?

Weird, but seems harmless enough

Reply:
actually, it looks like they make the id's into `unique=false`, which seems like we wouldn't want that?

old:
```
Indexes:
    "data_categories_pkey" PRIMARY KEY, btree (id)
    "ix_data_categories_fides_key" UNIQUE, btree (fides_key)
    "ix_data_categories_id" UNIQUE, btree (id)
```

new:
```
Indexes:
    "data_categories_pkey" PRIMARY KEY, btree (id)
    "ix_data_categories_fides_key" UNIQUE, btree (fides_key)
    "ix_data_categories_id" btree (id)
```

Reply:
@sanders41 confirmed this was probably left out of the [migration to change to fideslib](https://github.com/ethyca/fides/blob/main/src/fidesctl/api/ctl/migrations/versions/155fd8e51d9d_add_fideslib_models.py). The behavior is the same, as fideslib added a `primary_key` which is by default already unique, so this should be a safe change 👍